team.py

```diff
@@ -40,4 +42,20 @@ def edit_password(self, input):
             return False
         self.password = password
         ret = "Password successfully changed to " + password
-        return ret
\ No newline at end of file
+        return ret
+
+    def answer_question(self, user_answer):
+        if self.database.get_current_user() is not self:
+            return "Cannot answer question when not logged in!"
+        landmarks = self.database.get_landmark_path()
+        try:
+            check_landmark = landmarks[self.current_landmark]
+        except IndexError:
+            return "Not at a valid landmark"
+        ret_string = ""
+        if check_landmark.verify_answer(user_answer):
```

Comment:
if check_landmark was created in the try block, can it still be accessed here? I don't know how the scope works in python so this may be valid, but I figured I'd ask